generator/templates/client/parameter.gotmpl

```diff
@@ -91,7 +91,7 @@ type {{ pascalize .Name }}Params struct {
   {{ blockcomment .Description }}
 
   {{ end }}*/
-  {{ pascalize .ID }} {{ if and (not .IsArray) (not .IsMap) (not .HasDiscriminator) (not .IsInterface) (not .IsStream) (or .IsNullable  ) }}*{{ end }}{{ if not .IsFileParam }}{{ .GoType }}{{ else }}runtime.NamedReadCloser{{end}}
+  {{ or (index .Extensions "x-go-name") (pascalize .ID) }} {{ if and (not .IsArray) (not .IsMap) (not .HasDiscriminator) (not .IsInterface) (not .IsStream) (or .IsNullable  ) }}*{{ end }}{{ if not .IsFileParam }}{{ .GoType }}{{ else }}runtime.NamedReadCloser{{end}}
```

Comment:
@maxatome, there is also a templates/server/parameter.gotmpl

Overall, I am not convinced of the template-based approach to alias parameters. Have you thought about changing it here instead? https://github.com/go-swagger/go-swagger/blob/1c98855b472d8782c366459428ec2a5e8339ccf2/generator/operation.go#L741

Reply:
You are right it is a way simpler :)
Done.